launcher/test-framework/org.jboss.tools.common.launcher.reddeer/src/org/jboss/tools/common/launcher/reddeer/wizards/NewLauncherProjectWizardPage.java

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.launcher.reddeer.wizards;
+
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.swt.api.Button;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.combo.DefaultCombo;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.eclipse.swt.widgets.Composite;
+
+
+/**
+ * 
+ * @author zcervink
+ *
+ */
+public class NewLauncherProjectWizardPage extends WizardPage {
```

Comment:
I am afraid that you are extending eclipse.jface wizarr page class instead of reddeer's wizard page. Please use org.eclipse.reddeer.jface.wizard.WizardPage. Also, this would mean that finish methods would move up into WizardDialog that actually encapsulates wizard page object in general. See comment above.

Reply:
Great, thanks.